pkg/apply/applier_test.go

```diff
@@ -308,28 +324,32 @@ func TestValidateNamespace(t *testing.T) {
 			objects: []*resource.Info{obj1Info, obj2Info},
 			isValid: true,
 		},
-		"Three resources with same namespace is valid": {
-			objects: []*resource.Info{obj1Info, obj2Info, obj3Info},
+		"Two resources with same namespace and cluster-scoped obj is valid": {
```

Comment:
Out of curiosity, why is namespace-scoped + cluster-scoped supported but not multiple namespaces? Given the extent of the blast-radius difference, in Krane we decided to not only ban these types from being created together, but even forced a different command to be used. In our context, cluster-scoped resources are generally created by a different type of user than namespace-scoped resources, but even in the case where a given user needs to create both, this separation gives a stronger guarantee that they're aware of what they're doing and not unintentionally polluting the global scope with a resource they believed to be namespaced (for example). It definitely reduces convenience for some so I understand it might not be appropriate here, but curious about your reasoning.